athena/tests/test_util_functions.py

```
@@ -0,0 +1,64 @@
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)